### DISCUSSION TITLE
Trick to getting it to show up in Community?

### DISCUSSION THREAD

**Discussion by xenorim** [lloesche/valheim-server-docker#34] [Q&A]
Has anyone figured out the trick to getting your dedicated server to show up in the Community list? I can connect to it fine directly through the Steam server list, both internally and externally.

**Comment by n00del** [comment]
I have the same problem with my server.
Would be a nice feature to use the ingame browser to join instread of using steam server browser.

**Reply by xenorim** [reply]
I submitted that as feedback via the option they have in game.  Maybe we will see it soon if enough people request it

**Comment by lloesche** [comment]
It used to show up in-game just fine. But ever since the list of servers has grown > 5000 new servers don't seem to show up. I've had the same when starting the native Windows server, so the issue doesn't seem to be Linux or container related.

**Reply by xenorim** [reply]
Ah got ya.   Thanks for the quick reply.
